pkgs/development/libraries/qtpbfimageplugin/default.nix

```diff
@@ -23,9 +23,15 @@ stdenv.mkDerivation rec {
   '' + lib.optionalString stdenv.isDarwin ''
     # Fix darwin build
     substituteInPlace pbfplugin.pro \
+      --replace '$$PROTOBUF/include' '${protobuf}/include' \
       --replace '$$PROTOBUF/lib/libprotobuf-lite.a' '${protobuf}/lib/libprotobuf-lite.dylib'
   '';
 
+  qmakeFlags = lib.optionals stdenv.isDarwin [
+    "CONFIG+=c++17"
+    "QMAKE_MACOSX_DEPLOYMENT_TARGET=10.15"
```

Comment:
Should we get this from the sdk version?

Reply:
I don't think so, it's just to avoid an error (in the comment) mentioning macOS 10.15. Also, this package seems to rely on Xcode Command Line Tools. Should we mark this as broken for now?

Reply:
If it doesn't work sure. We can always revisit it. Also this keeps the scope of this PR manageable.